src/core/Akka.Remote/Transport/Helios/HeliosTcpTransport.cs

```diff
@@ -107,8 +105,17 @@ public override void ChannelRead(IChannelHandlerContext context, object message)
         /// <param name="exception">TBD</param>
         public override void ExceptionCaught(IChannelHandlerContext context, Exception exception)
```

Comment:
If we shutdown while read operation is in progress, SocketException with SocketError.OperationAborted code is expected, hence we know the exact reason of the Disassociation -Shutdown.

The rest - like before - we log the exception and signal Disassociation - unknown.
Here we log exception only if we don't know what it's and the